From 46d2433c7e8e8b89352c823a6ccf267d3ac7882b Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@freescale.com>
Date: Fri, 3 Jul 2015 14:10:50 +0800
Subject: [PATCH] MLK-11123-25 ARM: dts: disable otg features for
 imx6sx-17x17-arm2 board

Disable all otg features for otg1 port of imx6sx-17x17-arm2 board, then
the otg port can only do role swtich by ID pin state.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 936c7a6..948c93a 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -497,6 +497,9 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usbotg1_1>;
 	disable-over-current;
+	srp-disable;
+	hnp-disable;
+	adp-disable;
 	status = "okay";
 };
 
-- 
1.8.0

